tests/python/pants_test/init/test_util.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import os
+
+from pants.testutil.pants_run_integration_test import PantsRunIntegrationTest
+from pants.util.contextutil import temporary_dir
+from pants.util.dirutil import safe_mkdir
+
+
+class TestUtil(PantsRunIntegrationTest):
```

Comment:
Thanks for adding a test @patliu85. As we discussed in private we can test the init_workdir with a unit test based on TestBase, which requires much less overhead than an integration test. Here is an example of how I did that: https://github.com/pantsbuild/pants/commit/6c6ed541bd35d78e8c515e1bde70f9be7b666077#diff-eec0fb0e15b5eb829b521bcee9047ca1R1-R47.  Can you modify this test to be a unit test?

Reply:
Thanks @hrfuller, I updated `tests/python/pants_test/init/test_util.py` with your changes.